src/SDKs/Reservations/Microsoft.Azure.Management.Reservations/generate.cmd

```diff
@@ -0,0 +1,7 @@
+::
+:: Microsoft Azure SDK for Net - Generate library code
+:: Copyright (C) Microsoft Corporation. All Rights Reserved.
+::
+
+@echo off
+call %~dp0..\..\..\..\tools\generate.cmd compute/resource-manager %*
```

Comment:
@corquiri the path should be according to your RP path in the spec repo

Reply:
corrected